dashboard/apps/moj_admin/templates/moj_admin/base.html

```diff
@@ -0,0 +1,114 @@
+{% extends 'moj_admin/moj_base.html' %}
+
+{% load staticfiles %}
+
+{% block top_of_page %}
+    {% load i18n admin_static %}
+{% endblock %}
+
+{% block page_title %}
+
+    MoJ Product Dashboard
+
+{% endblock %}
+
+{% block stylesheets %}
+
+    <link rel="stylesheet" type="text/css" href="{% static "moj_admin/css/dwp_admin.css" %}"/>
+
+    {% block extrastyle %}
+
+    {% endblock %}
+{% endblock %}
+
+{% block head %}
+    {% block blockbots %}
+        <meta name="robots" content="NONE,NOARCHIVE"/>
+    {% endblock %}
+    <script type="text/javascript" src="https://code.jquery.com/jquery-2.2.4.min.js"></script>
```

Comment:
yes, we should just need one. the one on `moj_base` is very old. we need to do something with it.